geoips/plugins/modules/readers/wfabba_ascii.py

```diff
@@ -18,6 +18,7 @@
 interface = "readers"
 family = "standard"
 name = "wfabba_ascii"
+source_names = ["wfabba"]
```

Comment:
Would you please update this to:
`["abi", "ahi", "seviri"]`

WFABBA is a fire product run by SSEC, and produces files for those sensors on geostationary satellites. Technically "gvar" could also be included (since it was produced for old goes when they were operational), but perhaps we should limit to active sensors for now? What are your thoughts @mindyls?

Reply:
I added 'gvar' just in case. Should be good now but I'm happy to remove that in the case we only want to support active sensors. Thanks for reviewing!